data/pc/1.16.1/blockCollisionShapes.json

```diff
@@ -0,0 +1,1938 @@
+        "andesite_wall": [125, 126, 125, 126, 127, 128, 127, 128, 129, 130, 129, 130, 131, 132, 131, 132, 133, 134, 133, 134, 135, 136, 135, 136, 137, 138, 137, 138, 139, 140, 139, 140, 141, 142, 141, 142, 143, 144, 143, 144, 145, 146, 145, 146, 147, 148, 147, 148, 149, 150, 149, 150, 151, 152, 151, 152, 153, 124, 153, 124, 154, 155, 154, 155],
+        "red_nether_brick_wall": [125, 126, 125, 126, 127, 128, 127, 128, 129, 130, 129, 130, 131, 132, 131, 132, 133, 134, 133, 134, 135, 136, 135, 136, 137, 138, 137, 138, 139, 140, 139, 140, 141, 142, 141, 142, 143, 144, 143, 144, 145, 146, 145, 146, 147, 148, 147, 148, 149, 150, 149, 150, 151, 152, 151, 152, 153, 124, 153, 124, 154, 155, 154, 155],
+        "sandstone_wall": [125, 126, 125, 126, 127, 128, 127, 128, 129, 130, 129, 130, 131, 132, 131, 132, 133, 134, 133, 134, 135, 136, 135, 136, 137, 138, 137, 138, 139, 140, 139, 140, 141, 142, 141, 142, 143, 144, 143, 144, 145, 146, 145, 146, 147, 148, 147, 148, 149, 150, 149, 150, 151, 152, 151, 152, 153, 124, 153, 124, 154, 155, 154, 155],
+        "end_stone_brick_wall": [125, 126, 125, 126, 127, 128, 127, 128, 129, 130, 129, 130, 131, 132, 131, 132, 133, 134, 133, 134, 135, 136, 135, 136, 137, 138, 137, 138, 139, 140, 139, 140, 141, 142, 141, 142, 143, 144, 143, 144, 145, 146, 145, 146, 147, 148, 147, 148, 149, 150, 149, 150, 151, 152, 151, 152, 153, 124, 153, 124, 154, 155, 154, 155],
+        "diorite_wall": [125, 126, 125, 126, 127, 128, 127, 128, 129, 130, 129, 130, 131, 132, 131, 132, 133, 134, 133, 134, 135, 136, 135, 136, 137, 138, 137, 138, 139, 140, 139, 140, 141, 142, 141, 142, 143, 144, 143, 144, 145, 146, 145, 146, 147, 148, 147, 148, 149, 150, 149, 150, 151, 152, 151, 152, 153, 124, 153, 124, 154, 155, 154, 155],
+        "scaffolding": 246,
+        "loom": 1,
+        "barrel": 1,
+        "smoker": 1,
+        "blast_furnace": 1,
+        "cartography_table": 1,
+        "fletching_table": 1,
+        "grindstone": [248, 248, 249, 249, 247, 250, 251, 252, 253, 253, 254, 254],
+        "lectern": 255,
+        "smithing_table": 1,
+        "stonecutter": 256,
+        "bell": [257, 257, 257, 257, 258, 258, 258, 258, 259, 259, 259, 259, 259, 259, 259, 259, 260, 260, 261, 261, 262, 262, 263, 263, 264, 264, 264, 264, 265, 265, 265, 265],
+        "lantern": [267, 266],
+        "campfire": 268,
+        "sweet_berry_bush": 0,
+        "structure_block": 1,
+        "jigsaw": 1,
+        "composter": 269,
+        "bee_nest": 1,
+        "beehive": 1,
+        "honey_block": 64,
+        "honeycomb_block": 1,
+        "nether_gold_ore": 1,
+        "soul_fire": 0,
+        "soul_soil": 1,
+        "basalt": 1,
+        "polished_basalt": 1,
+        "soul_torch": 0,
+        "soul_wall_torch": 0,
+        "chain": 270,
```

Comment:
The shapes array stop at 269. Could you add an entry for the chain block ?

Reply:
I did at the bottom didn't I?

Reply:
Bruh it disappeared

Reply:
thanks